cdist/conf/type/__apt_default_release/man.rst

```diff
@@ -0,0 +1,44 @@
+cdist-type__apt_default_release(7)
+==================================
+
+NAME
+----
+cdist-type__apt_default_release - Configure the default release for apt
+
+
+DESCRIPTION
+-----------
+Configure the default release for apt, using the APT::Default-Release
+configuration value. The name of the object sets the default release
+and can contain release name, codename or release version. Examples:
+'stable', 'testing', 'unstable', 'stretch', 'buster', '4.0', '5.0*'.
+
+REQUIRED PARAMETERS
+-------------------
+None.
```

Comment:
In description states that object name is release value and here is no parameter mentioned, but in the code you use release parameter and is in the list of required parameters. Can you fix this inconsistency?

Reply:
Ah, I originally wanted to use the object name, but didn't know a singleton has no name. I probably forgot to update the manpage afterwards.

In any case, I fixed this now. I added a fixup commit instead of amending the previous commit, for easier review, but they should be squashed before merging (but I think you usually manually merge anyway, right?).